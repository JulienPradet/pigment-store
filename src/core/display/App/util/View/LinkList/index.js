import React from 'react'
import styles from './link-list.m.css'

export const Container = ({children}) => <div className={styles.container}>{children}</div>
Container.displayName = 'LinkList.Container'

export const Title = ({children}) => <div className={styles.title}>{children}</div>
Title.displayName = 'LinkList.Title'

export const Content = ({children}) => <div className={styles.content}>{children}</div>
Content.displayName = 'LinkList.Content'
