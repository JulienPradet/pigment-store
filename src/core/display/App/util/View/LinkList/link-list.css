.container {
}

.title {
  color: #a6cf39;
  font-weight: bold;
}

.content a {
  color: #39354b;
  text-decoration: none;
}

.content a:hover {
  text-decoration: underline;
}
