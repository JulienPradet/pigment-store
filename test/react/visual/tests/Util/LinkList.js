import React from 'react'
import PigmentStore from '../../../../../dist/index'
import {Container, Title, Content} from '../../../../../src/core/display/App/util/View/LinkList/index'

export default PigmentStore.React.describe('LinkList', Container)
  .featureJsx(
    'Default',
    <Container>
      <Title>Basic link list</Title>
      <Content>
        <ul>
          <li><a href='/'>Link 1</a></li>
          <li><a href='/'>Link 2</a></li>
          <li><a href='/'>Link 3</a></li>
        </ul>
      </Content>
    </Container>
  )
