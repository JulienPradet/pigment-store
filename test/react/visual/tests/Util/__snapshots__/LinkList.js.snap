exports[`LinkList Default 1`] = `
<div
  className="container">
  <div
    className="title">
    Basic link list
  </div>
  <div
    className="content">
    <ul>
      <li>
        <a
          href="/">
          Link 1
        </a>
      </li>
      <li>
        <a
          href="/">
          Link 2
        </a>
      </li>
      <li>
        <a
          href="/">
          Link 3
        </a>
      </li>
    </ul>
  </div>
</div>
`;
